<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>376</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MaSim Visualization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_sim_data">
       <attribute name="title">
        <string>Visualization</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignmentFlag::AlignHCenter">
           <widget class="QComboBox" name="cb_data_list">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_process">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
           <widget class="QPushButton" name="bt_run">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Run  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_progress">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TickPosition::NoTicks</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_4" rowstretch="2,2,0">
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,0">
              <item>
               <widget class="QChartView" name="gv_chartview">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>240</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="ColorMapWidget" name="wg_color_map" native="true">
              <layout class="QGridLayout" name="gridLayout_5"/>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="9">
              <item>
               <widget class="GraphicsViewCustom" name="graphicsView"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="le_sim_path">
              <property name="toolTip">
               <string>Input simulation path or use button to select folder</string>
              </property>
              <property name="accessibleName">
               <string/>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Type in simulation path then press Enter, or use Browse button</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bt_auto_load_folder">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_7" rowstretch="2,1">
          <item row="0" column="0">
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_submit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_graphs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QCharts</header>
  </customwidget>
  <customwidget>
   <class>GraphicsViewCustom</class>
   <extends>QGraphicsView</extends>
   <header>graphicsviewcustom.h</header>
  </customwidget>
  <customwidget>
   <class>ColorMapWidget</class>
   <extends>QWidget</extends>
   <header>colormapwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
